### PR TITLE
Support io.js

### PR DESCRIPTION
**Pull request by mhayashi**
Added support for io.js

### CODE REVIEW

**Comment by guilhem** [comment]
+1 for this feature
I just dislike ['nodejs']['prefix_url'] generation :/

Can you use ruby in attributes plz?

**Comment by guilhem** [comment]
We really need io.js support ;)

https://www.youtube.com/watch?v=1IOukA10QeQ

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs/Configures nodejs'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.2.1'
+version '2.2.2'
```
Comment (BarthV) [inline metadata.rb]: I don't like Version Bump in PR :)

**Comment by guilhem** [comment]
WIP in branch [iojs](https://github.com/redguide/nodejs/tree/iojs)

**Comment by mansona** [comment]
In the mean time you can check out an io.js specific cookbook here: https://github.com/Blooie/io.js-cookbook https://supermarket.chef.io/cookbooks/iojs

**Comment by guilhem** [comment]
@mansona yes but it's a really "simple" CK ;)

**Comment by BarthV** [comment]
I'll test io branch this morning

**Comment by mansona** [comment]
@guilhem :joy: yes indeed it is, happy to improve it if you have any feature requests! 

Sorry for trolling this issue, i'll stop now :trollface:

**Comment by guilhem** [comment]
superseded by #65

**Pull request closed without merging**